python/src/endpoints/db_nl_query.py

```
@@ -1,13 +1,53 @@
+        response = client.db.process_nl_query(
+            question="How many active users do we have?",
+            table="users",
+        )
 
-if __name__ == '__main__':
-    print(db_nl_query())
+        # Handle the successful response
+        print(f"Result: {response.data}")
+        print(f"Generated SQL: {response.sql}")
+        print(f"Query execution time: {response.execution_time} ms")
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)